build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts

```diff
@@ -16,7 +16,20 @@
 
 repositories {
     maven {
-        name = "Gradle libs"
-        url = uri("https://repo.gradle.org/gradle/libs")
+        name = "Gradle public repository"
+        url = uri("https://repo.gradle.org/gradle/public")
+        content {
+            includeGroup("net.rubygrapefruit")
+            includeModule("classycle", "classycle")
+            includeModule("flot", "flot")
+            includeModuleByRegex("org.gradle", "sample-(check|discovery)")
```

Comment:
I suppose this would go away after exemplar is on Maven Central, right?

Reply:
Yes.